app/controllers/users_controller.rb

```diff
@@ -0,0 +1,36 @@
+class UsersController < ApplicationController
+  
+  def index 
+    @users = User.all
+  end
+  
+  def show
+    user_id = params[:id]
+    @user = User.find_by(id: user_id)
```

Comment:
This would be good functionality to put in a controllor filter `find_user` in the Application Controller (so that all controllers inherit this method).